datareduction/datareductionoperator.cpp

```diff
@@ -1642,7 +1642,7 @@ namespace DRO {
          }
       }
 
-      # pragma omp parallel
+      #pragma omp parallel
```

Comment:
Can this  VariablePrecipitationLineDiffFlux also be ported to ARCH?

Reply:
> Can this VariablePrecipitationLineDiffFlux also be ported to ARCH?

I added arch for all three remaining loops now.